doc/content/demos/multi-user-chat/multi-user-chat.py

```diff
@@ -0,0 +1,205 @@
+from lona.html import (
+    TextInput,
+    TextArea,
+    Button,
+    Table,
+    Span,
+    HTML,
+    Div,
+    Ul,
+    Tr,
+    Td,
+    Li,
+    H1,
+    P,
+    A,
+)
+from lona import LonaView, LonaApp
+
+app = LonaApp(__file__)
+
+
+@app.route('/')
+class ChooseRoom(LonaView):
+    def handle_request(self, request):
+        # find rooms
+        rooms = set()
+
+        with self.server.state.lock:
+            if 'rooms' in self.server.state:
+                rooms = set(self.server.state['rooms'].keys())
+
+        # setup html
+        html = HTML(
+            H1('Choose A Chat Room'),
+
+            Ul(
+                list(Li(A(room, href=f'/{room}')) for room in rooms),  # NOQA: C400
+            ),
+
+            TextInput(
+                placeholder='Name',
+                _id='new-room-name',
+                bubble_up=True,
+            ),
+            Button('Create', _id='create-new-room'),
+        )
+
+        # choose a room
+        while True:
+            input_event = self.await_input_event(html=html)
+
+            # button
+            if input_event.node_has_id('new-room-name'):
+                new_room = input_event.node.value
+
+                if new_room in rooms:
```

Comment:
Couldn't this be simplified to a one-liner? `html.query_selector('#create-new-room').disabled = new_room in rooms`

Reply:
Yes that's right, but that's demo code, and i wanted it to be as verbose as possible

Reply:
@MShekow Do you agree, or should i change it?

Reply:
The verbose version is a bit easier to understand, so you might as well leave it as it is ;)